gobblin-yarn/src/test/java/org/apache/gobblin/yarn/YarnServiceTestWithExpiration.java

```diff
@@ -204,7 +204,7 @@ public void testStartError() throws Exception{
     Assert.assertEquals(this.expiredYarnService.getNumRequestedContainers(), 10);
 
     try {
-      Thread.sleep(20000);
+      Thread.sleep(60000);
```

Comment:
Usually it is not a good practice to use thread.sleep in the unit test if not necessary. (and 60 seconds is really long if you considering the number of unit tests we have). 

why is this needed ? is there any other way to achieve the same? There's a test utilities called `AssertWithBackoff` you might want to look into, if satisfying certain condition is what you want after the sleep.